bundles/ehcache/src/main/filtered-resources/OSGI-INF/bundle.info

```diff
@@ -0,0 +1,2 @@
+\u001B[1mSYNOPSIS\u001B[0m
+    ${project.description}
```

Comment:
FTR at least one file needs to exist as a source so that maven-bundle-plugin doesn't freakout about missing target/classes directory.  This is a standard karaf file to work with the {{bundle:info}} command.